cluster/examples/gcp-credentials.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
is this gcp-credentials.sh helper script being referenced anywhere in the docs? Looks like it does the same steps/flow that's manually documented in `gcp-provider.md`.

Should it be linked from that doc as an automated option for folks that don't want to copy/paste all the commands? and to help improve the discoverability of this script and to help users understand it's pre-reqs and usage?

Reply:
Sounds good to me. I'll add a couple of lines about in the Google credentials readme.

Reply:
Added in 50510cf